src/scripts/main.js

```diff
@@ -1,3 +1,20 @@
 'use strict';
 
-// write code here
+const allLi = document.querySelector('.tree').querySelectorAll('li');
```

Comment:
You can combine the two 'querySelector' calls into one for better performance. For example, you could use: 'document.querySelectorAll('.tree li')'.